bench/db_bench.ml

```diff
@@ -48,15 +48,23 @@ let populate () =
   in
   loop 0
 
-let print_results db f nb_entries =
+let print_results db f nb_entries operation =
   let _, time = with_timer f in
   let micros = time *. 1_000_000. in
   let sec_op = micros /. float_of_int nb_entries in
   let mb = float_of_int (entry_size * nb_entries / 1_000_000) /. time in
   let ops_sec = float_of_int nb_entries /. time in
   Log.app (fun l ->
-      l "%s: %f micros/op; \t %f op/s; \t %f MB/s; \t total time = %fs." db
-        sec_op ops_sec mb time)
+      l "%s %s: %f micros/op; \t %f op/s; \t %f MB/s; \t total time = %fs." db
+        operation sec_op ops_sec mb time);
```

Comment:
I'm not sure if you should print `operation` here as is something like `fill_random`, not very intelligible as a user output. Also we print the operation before doing it anyway.